crates/matrix_mentions/src/mention.rs

```diff
@@ -215,6 +217,19 @@ fn parse_external_id(uri: &str) -> Result<MatrixToUri, IdParseError> {
     MatrixToUri::parse(&uri_for_ruma)
 }
 
+/// Attempt to find `/#/user/` or `/#/room/` in the supplied URI, and split it on one of those if
+/// found, meaning it is a URI into a client like Element Web. Otherwise split it on `/#/`,
+/// treating it as if it were a matrix.to URI.
+fn split_uri_on_prefix(uri: &str) -> Vec<&str> {
```

Comment:
Since this function is only needed for custom Matrix URLs should it be annotated with the `feature` attribute?
```
#[cfg(any(test, feature = "custom-matrix-urls"))]
```

Reply:
Good spot, thank you. Fixed in 69823a65